yarn-project/end-to-end/src/simulators/lending_simulator.ts

```diff
@@ -26,7 +26,7 @@ export class LendingAccount {
    */
   public async key(): Promise<Fr> {
     return Fr.fromBuffer(
-      pedersenPlookupCommitInputs(
+      pedersenCompressInputs(
```

Comment:
This is done in many places, compressInputs is still a janky name -- I will change it in a separate PR to hash